sBTC/sbtc_signer.md

```diff
@@ -7,60 +7,48 @@
+Delegated signing enables designated signers to perform transactions on behalf of users, improving the efficiency and scalability of the signing process. This is a requirement for custodians if the institution chooses not to self-stack. 
 
-**Requirements:**
-
-| No. | Task                                                                                                                                   |
-| --- | -------------------------------------------------------------------------------------------------------------------------------------- |
-| 1   | Allow setting a maximum auto-approve transaction amount. Transactions exceeding this amount will trigger a manual review for approval. |
-| 2   | Implement a default time limit for the manual review process (e.g., 24 hours) to ensure timely action on flagged transactions.         |
-
-## Requirement 3: Auto Deny Configuration After X Blocks
-
-**Description:**
-
-The system must automatically deny transactions after a specified number of blocks if the signer is unable to sign within the timeframe.
 
 **Requirements:**
 
-| No. | Task                                                                                                                                                                  |
-| --- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| 1   | Allow configuration of the number of blocks after which a transaction is automatically denied.                                                                        |
-| 2   | Determine and specify the penalty mechanism for cases where a user does not sign a transaction during the manual review process (e.g., freezing or slashing rewards). |
+| No. | Task                                                                |
+| --- | ------------------------------------------------------------------- |
+| 1   | Implement delegated signing in V0.2 of the Signer product.                   |
+| 2   | Conduct further analysis of delegated signing product requirements. |
 
-## Requirement 4: Delegated Signing (Mini v2 Feature)
+## Requirement 3: Implement an Auto-Deny Configuration For Transactions That Require a Manual Review 
 
 **Description:**
 
-Delegated signing enables designated signers to perform transactions on behalf of users, improving the efficiency and scalability of the signing process.
+The system must be able to automatically deny transactions after a protocol-defined number of blocks if the Signer is unable to manually sign within the timeframe. The purpose of this requirement is to accommodate a Signer who has logic that triggers a manual review but is unable to perform the action in the defined timeframe. Rather than allow them to pass the vote deadline and potentially fail to sign, this would default sign Deny.
```

Comment:
`Rather than allow them to pass the vote deadline and potentially fail to sign, this would default sign Deny` reads oddly to me. 
maybe something like this?
```
By default, signers will Deny a transaction rather than allow them to pass the vote deadline and potentially fail to sign.
```